samples/serial/README.md

```diff
@@ -0,0 +1,150 @@
+    - Host connection information (IP address and TCP port number).
+
+- For server:
+    - Software to listen to the pre-assigned port and establish a connection to the client.
+    - Relay the communications to/from the client with the serial device through a physical serial port.
+
+
+## Configure serial over network on EFLOW
+
+Use socat on the client, we configure socat to create a pseudo-ttyS as the device and associate the pseudo-ttyS device with a network port. A pseudo-tty is like a serial port in that it has a /dev entry that can be opened by a program that expects a serial port device, except that instead of belonging to a physical serial device, the data can be intercepted by another program. The socat program intercepts the traffic to/from this pseudo-ttyS and relays the network port data.
+
+Use hub4com from the [com0com project](https://sourceforge.net/projects/com0com/files/hub4com/2.1.0.0/) on the server-side. We setup hub4com to associate the network port with the physical serial port, hub4com listens to the assigned port, establishes a connection to the client, and relays data from network port to physical port.
+
+Here is an example EFLOW configuration
+
+- Host ip = **172.18.246.137**
+- Host tcp port = **5002**
+- Host physical serial port = **COM3**
+
+- Client virtual serial port name = **/dev/ttyVirtS0**
+
+
+### Setup Host
+- Download and extract hub4com-2.1.0.0-386.zip from [com0com project](https://sourceforge.net/projects/com0com/files/hub4com/2.1.0.0/) to a local directory, e.g. c:\hub4com
+- Pick a TCP port number to associate to the serial port planned to expose.  In our example environment, we use port number 5002, and the port is associated with COM3
+    > If you have a firewall enabled, explicitly configure the firewall rule to allow inbound traffic for TCP port 5002.  Or run the command in the next step and allow access when asked.
+- Run hub4com in server mode, open a command prompt, type `cd /d c:\hub4com` and `com2tcp.bat \\.\COM3 5002` where **COM3** is the physical serial port name and **5002** is the TCP port number.  Click 'Allow access' if the Windows Defender Firewall dialog box pops up and asks for port access.
+- Hub4Com starts in server mode waiting for a connection.
+	
+
+### Setup Client
+- Connect EFLOW VM via Windows Admin Center or Powershell.  Check [Create a new deployment](https://docs.microsoft.com/en-us/azure/iot-edge/how-to-install-iot-edge-on-windows?view=iotedge-2018-06&tabs=powershell#create-a-new-deployment) on how to create your deployment of EFLOW on your target device.
+- Run socat in client mode: from the WAC CLI or Powershell window, type `sudo socat pty,link=/dev/ttyVirtS0,raw,user=iotedge-user,group=dialout,mode=777 tcp:172.18.246.137:5002`.  The command create a virtual serial port (**/dev/ttyVirtS0**) and relay data between the virtual serial port and server (ip address **172.18.246.137**, port **5002**)
```

Comment:
**This** command **creates** a virtual serial port (**/dev/ttyVirtS0**) and **relays** data between the virtual serial port and server (**IP** address **172.18.246.137**, port **5002**) **.**

Reply:
fixed